src/LanguageServer/Protocol/Features/Diagnostics/EngineV2/DiagnosticIncrementalAnalyzer.HostAnalyzerInfo.cs

```diff
@@ -145,9 +144,8 @@ private static int GetPriority(DiagnosticAnalyzer state)
 
             return state switch
             {
-                FileContentLoadAnalyzer _ => FileContentLoadAnalyzerPriority,
                 GeneratorDiagnosticsPlaceholderAnalyzer _ => GeneratorDiagnosticsPlaceholderAnalyzerPriority,
-                DocumentDiagnosticAnalyzer analyzer => Math.Max(0, analyzer.Priority),
+                DocumentDiagnosticAnalyzer analyzer => analyzer.Priority,
```

Comment:
removed as we do just want to pass the priority through.  Note that FileContentLoad is still pri -4, and the rest of the doc  analyzers (which we own 100%) are all just at 50 and never change.